docs/GettingStarted.md

```diff
@@ -123,8 +122,16 @@ architecture (powerpc or amd64) from the website and run it.
         ...
 
 
+Note: 
+	1)If there is diffrent OS running on the swicth already.Then stop it at U-boot mode,
+	      => printenv
+           check the onie details.Open the ONIE in rescue mode by,
+	      => run onie_rescue
+           It will take you to the ONIE environment.
```

Comment:
I would add a line that says "Note: while ONIE has many different installation modes, we recommend the rescue mode for doing a manual (read: via console) because it disables the automatic ONIE server discovery"